docs/details/shards/General/Await.md

```diff
@@ -0,0 +1,6 @@
+`(Await)` triggers the execution of its shard (or sequence of shards) and passes the control back to the main program. Once the `(Await)` shard(s) have finished their execution the result of their computation is made available to the prgoram by `(Await)`.
```

Comment:
Typo "prgoram"

Reply:
Await actually runs the inner shards in a task that is sent to the thread pool specifically

Reply:
Fixed/rewritten.